core/os/device/gpu_counter_descriptor.proto

```diff
@@ -0,0 +1,183 @@
+// Copyright (C) 2019 Google Inc.
```

Comment:
Isn't it possible for a counter to have none of those units? Or combined units (In fact I expect MANY of our counters to have non-standard units).

#Vertices #Fragments MB/s ... etc?

Reply:
maybe, unit here is optional
In fact the current returned unit from prototyped producer is completely wrong. If we need more we can create a request to perfetto and then sync from perfetto.

Reply:
We need to fix this in Perfetto now. I really, really don't want to handle all these units in the UI.

Reply:
I am confused, which part are you referring to?

Reply:
This enum is copied from Perfetto, no? We should fix the Perfetto proto and drop all the non-nonsensical units and add units as Andrew mentioned (Vertices, Fragments, etc.)

Reply:
aosp/1107601